docker-entrypoint.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+export CONFIG_DIR="${CONFIG_DIR:-/opt/config}"
+export CONFIG=$CONFIG_DIR/config.json
+export FORCE_BUILD_BEFORE_START="${FORCE_BUILD_BEFORE_START:-false}"
+
+# Supported verbs: install (web install), setup (interactive CLI session). Default: web install
+# TODO: constraint it using a hash set (or hash table)
+export NODEBB_INIT_VERB="${NODEBB_INIT_VERB:-install}"
+# Setup variable for backward compatibility, default: <empty>
+export SETUP="${SETUP:-}"
+
+mkdir -p $CONFIG_DIR
+chmod -fR 760 $CONFIG_DIR
+
+if [[ ! -w $CONFIG_DIR/package.json ]]; then
```

Comment:
This actually returns `true` if `$CONFIG_DIR/package.json` doesn't exist. You need to check the directory itself here.

Fixing this resolves my issue from https://github.com/NodeBB/NodeBB/pull/8704#issuecomment-1195899536 - though `chmod` still throws an error